### PR TITLE
Add API documentation.

### PR DESCRIPTION
**Pull request by dan-zheng**
Document `PythonConvertible` and `ConvertibleFromPython`.

### CODE REVIEW

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please clean test tensorflow

File: stdlib/public/Python/Python.swift [hunk 1]
```diff
@@ -146,23 +146,12 @@ extension PythonObject : CustomReflectable {
 // `PythonConvertible` protocol
 //===----------------------------------------------------------------------===//
 
+/// A type that can be converted to a `PythonObject`.
```
Comment (rxwei) [inline stdlib/public/Python/Python.swift]: This is incorrect. 
```suggestion
/// A type whose values can be converted to a `PythonObject`.
```

File: stdlib/public/Python/Python.swift [hunk 2]
```diff
@@ -182,6 +171,19 @@ fileprivate extension PythonConvertible {
   }
 }
 
+//===----------------------------------------------------------------------===//
+// `ConvertibleFromPython` protocol
+//===----------------------------------------------------------------------===//
+
+/// A type that can be initialized from a `PythonObject`.
```
Comment (rxwei) [inline stdlib/public/Python/Python.swift]:
```suggestion
/// A type whose values can be initialized from a `PythonObject`.
```

Comment (dan-zheng) [reply stdlib/public/Python/Python.swift]: Discussed in person; the existing wording is actually okay.

**Review by rxwei — APPROVED** [review]

**Pull request merged**